app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -64,11 +67,12 @@ class TopicFragmentPresenter @Inject constructor(
     binding.topicToolbar.setNavigationOnClickListener {
       (activity as TopicActivity).finish()
     }
-
-    binding.topicToolbar.setOnClickListener {
-      binding.topicToolbarTitle.isSelected = true
+    val screenReaderEnabled = accessibilityService.isScreenReaderEnabled()
```

Comment:
I can't see any usages for this variable. Please let me know if I'm missing something.